pkg/types/ovirt/platform.go

```diff
@@ -33,4 +33,16 @@ type Platform struct {
 	// Default will set the image field to the latest RHCOS image.
 	// +optional
 	DefaultMachinePlatform *MachinePool `json:"defaultMachinePlatform,omitempty"`
+
+	// AffinityGroupsNames contains the names of the RHV affinity groups that the machines will join
```

Comment:
This is not the correct name of the field.
Field descriptions should end in periods.
```
	// AffinityGroups is the list RHV affinity groups that the machines will join.
```

Reply:
Done